lib/middleware/schemas/templates.schema.json

```diff
@@ -174,9 +180,36 @@
               "type": "string"
             }
           }
+        },
+        "introduction": {
+          "description": "Form introduction",
+          "id": "#/properties/introduction",
+          "type": "object",
+          "properties": {
+            "descriptionEn": {
+              "type": "string"
+            },
+            "descriptionFr": {
+              "type": "string"
+            }
+          }
+        },
+        "privacyPolicy": {
+          "description": "Form privacy policy",
+          "id": "#/properties/privacyPolicy",
+          "type": "object",
+          "properties": {
+            "descriptionEn": {
+              "type": "string"
+            },
+            "descriptionFr": {
+              "type": "string"
+            }
+          }
         }
       },
-      "required": ["elements", "layout", "titleEn", "titleFr"]
+      "required": ["elements", "layout", "titleEn", "titleFr"],
```

Comment:
Should this match the publish requirements as far as fields go?

If we we need to add privacyPolicy and endPage

<img width="800" alt="Screen Shot 2023-01-06 at 9 13 25 AM" src="https://user-images.githubusercontent.com/62242/211029271-648d77ed-2932-46cc-bd48-226944dcbf80.png">

Reply:
I guess it would still work even if a user does not add a privacy statement/form confirmation message because you would send empty strings when saving a draft? I will add them in there.

Reply:
Tested it locally to validate that assumption 👍